leoserver.py passes an incorrect argument to websockets.exceptions.ConnectionClosed

Rev 868a03d suppresses the mypy warnings (see below) pending resolution of this issue.

`ws_handler` (the server main loop) contains this code:

```python
except TerminateServer as e:
    # pylint: disable=no-value-for-parameter,unexpected-keyword-arg
    raise websockets.exceptions.ConnectionClosed(code=1000, reason=e.__str__())
```

As you can see, pylint has always complained about this call.

Recent versions of mypy now also complain:
```console
mypy-leo
leo\core\leoserver.py:5450: error: Unexpected keyword argument "code" for "ConnectionClosed"  [call-arg]
C:\Python\Python3.11\Lib\site-packages\websockets\exceptions.py:91: note: "ConnectionClosed" defined here
leo\core\leoserver.py:5450: error: Unexpected keyword argument "reason" for "ConnectionClosed"  [call-arg]
C:\Python\Python3.11\Lib\site-packages\websockets\exceptions.py:91: note: "ConnectionClosed" defined here
```
It's time to fix this quirp. [Here](https://websockets.readthedocs.io/en/stable/reference/exceptions.html#websockets.exceptions.ConnectionClosed) is a documentation for `ConnectionClosed`.

--- a/leo/core/leoserver.py
+++ b/leo/core/leoserver.py
@@ -5446,7 +5446,7 @@ def main() -> None:  # pragma: no cover (tested in client)
                         print(f"{tag}: got: {d}", flush=True)
                     answer = controller._do_message(d)
                 except TerminateServer as e:
-                    raise websockets.exceptions.ConnectionClosed(1000, e.__str__())
+                    raise websockets.exceptions.ConnectionClosed()
                 except ServerError as e:
                     data = f"{d}" if d else f"json syntax error: {json_message!r}"
                     error = f"{tag}:  ServerError: {e}...\n{tag}:  {data}"

--- a/leo/core/leoserver.py
+++ b/leo/core/leoserver.py
@@ -5446,8 +5446,7 @@ def main() -> None:  # pragma: no cover (tested in client)
                         print(f"{tag}: got: {d}", flush=True)
                     answer = controller._do_message(d)
                 except TerminateServer as e:
-                    # pylint: disable=no-value-for-parameter,unexpected-keyword-arg
-                    raise websockets.exceptions.ConnectionClosed(code=1000, reason=e.__str__())  # type:ignore
+                    raise websockets.exceptions.ConnectionClosed(1000, e.__str__())
                 except ServerError as e:
                     data = f"{d}" if d else f"json syntax error: {json_message!r}"
                     error = f"{tag}:  ServerError: {e}...\n{tag}:  {data}"

